data/map planets.txt

```diff
@@ -3888,7 +3888,7 @@ planet Thule
 		fleet "Large Southern Pirates" 16
 
 planet Thunder
-	attributes rim
+	attributes rim forest
```

Comment:
What's the basis for marking Thunder as forst?

Reply:
Fair enough about the frontier jobs.

A perfect world would include a variety of native forests for the settlers to use. Also it's landing picture. Thunder is a boring planet too, I wanted to give it just a little bit more stuff.